enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -414,17 +414,52 @@ The list of connected hubs should now reflect the disabled state.
 
 Uninstalling Federated Reporting from a superhub is not possible at this time.
 
-In order to remove Federated Reporting from a feeder remove the ```cftransport``` user
-and ```/opt/cfengine/federation``` directories. This will remove trust established
-with the superhub and will prevent superhub from performing any dump/import procedures.
+In order to remove Federated Reporting from a feeder you must set the target_state
+to "off". On the next agent run the cftransport user will be removed, thus removing
+the trust established with the superhub and no further dump/import procedures will
+occur.
+
+There are two ways to change the target_state of a feeder.
+
+- Use the hub-state API (requires version 3.14.0 or greater on the feeder hub)
+- Edit federation-config.json (any version)
+
+### Uninstall with the API
+
+```console
+$ cat > target-state-off.json
+{
+  "target_state": "off"
+}
+```
 
 ```console
-# userdel -r cftransport
-# rm -rf /opt/cfengine/federation
+$ curl -k -i -s -X POST -u admin:$PASSWORD https://$FEEDER/api/fr/hub-state -d @target-state-off.json --header "Content-Type: application/json"
```

Comment:
`$USER` ? there are fine grained perms for this right?

Reply:
so $USER as opposed to admin? Seems like admin is the pattern so far for all the other examples here.

Reply:
```
     * @rbacName Change hub target state
     * @rbacGroup Federation report API
     * @rbacAlias hubState.post
```

Reply:
sure, that's fine.